templates/zerver/invalid_email.html

```diff
@@ -5,46 +5,31 @@
 {% endblock %}
 
 {% block portico_content %}
-
-    <h3>{{ _('Invalid email') }}</h3>
-
-    <p>{{ _('Hi there! Thank you for your interest in Zulip.') }}</p>
-
-    {% if invalid_email %}
-    <p>
-        {% trans %}
-        The email address you are trying to sign up with is not valid.
-        Please sign up using a valid email address.
-        {% endtrans %}
-    </p>
-    {% endif %}
-
-    {% if closed_domain %}
-    <p>
-        {% trans %}
-        The organization you are trying to join, {{ realm_name }},
-        only allows users with email addresses within the
-        organization. Please sign up using appropriate email address.
-        {% endtrans %}
-    </p>
-    {% endif %}
-
-    {% if disposable_emails_not_allowed %}
-    <p>
-        {% trans %}The organization you are trying to join,
-        {{realm_name}}, does not allow signups using disposable email
-        addresses. Please sign up using a real email address.
-        {% endtrans %}
-    </p>
-    {% endif %}
-
-    {% if email_contains_plus %}
-    <p>
-        {% trans %}The organization you are trying to join,
-        {{realm_name}}, does not allow signups using emails
-        that contains +. Please sign up using appropriate email address.
-        {% endtrans %}
-    </p>
-    {% endif %}
-
+<div class="app portico-page">
+    <div class="app-main portico-page-container center-block flex full-page account-creation new-style">
+        <div class="inline-block">
+            <div class="get-started">
+                <h1>{{ _("Invalid email") }}</h1>
+            </div>
+            <div class="white-box">
+                <p>{{ _("Hi there! Thank you for your interest in Zulip.") }}</p>
+                <p>
+                    {% if invalid_email %}
+                        {{ _("The email address you are trying to sign up with is not valid.") }}
+                    {% endif %}
+                    {% if closed_domain %}
+                        {% trans %}The organization you are trying to join, <a href="{{ realm_uri }}">{{ realm_name }}</a>, only allows users with email addresses within the organization.{% endtrans %}
+                    {% endif %}
+                    {% if disposable_emails_not_allowed %}
+                        {% trans %}The organization you are trying to join, <a href="{{ realm_uri }}">{{ realm_name }}</a>, does not allow signups using disposable email addresses.{% endtrans %}
+                    {% endif %}
+                    {% if email_contains_plus %}
+                        {% trans %}The organization you are trying to join, <a href="{{ realm_uri }}">{{ realm_name }}</a>, does not allow signups using emails that contain "+".{% endtrans %}
+                    {% endif %}
+                    {{ _("Please sign up using a valid email address.") }}
```

Comment:
So the `+` case especially, but really all of these feel awkward with "Please sign up using a valid email address." after it -- they're absolutely valid email addresses (according to the RFC, for example), just not ones that the administrators of this organization wants to allow to join. Maybe "using an allowed email address." would be OK?

Reply:
That tweak is fine by me. But then we should probably change the heading of those errors to something like "Email not allowed", rather than "Invalid email".

Reply:
Yeah, "Email not allowed" sounds good to me.